src/containers/DeveloperPortal/DeveloperPortalCards/index.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+import {Icon, IconType} from '@thenewboston/ui';
+import {Button, Container} from 'components';
+import {useHistory} from 'react-router';
+import ArchitectureIcon from '../icons/ArchitectureIcon';
+import ProjectsIcon from '../icons/ProjectsIcon';
+import './DeveloperPortalCards.scss';
+
+export default function DeveloperPortalCards() {
```

Comment:
I think we should switch these components to arrow functions to stay consistent with the rest of the codebase.